clang/test/dpct/cublaslt.cu

```diff
@@ -220,9 +220,13 @@ void foo3() {
   // CHECK: dpct::blas_gemm::experimental::epilogue_t e;
   // CHECK-NEXT: e = dpct::blas_gemm::experimental::epilogue_t::nop;
   // CHECK-NEXT: e = dpct::blas_gemm::experimental::epilogue_t::relu;
+  // CHECK-NEXT: e = dpct::blas_gemm::experimental::epilogue_t::gelu;
+  // CHECK-NEXT: e = dpct::blas_gemm::experimental::epilogue_t::gelu;
   cublasLtEpilogue_t e;
   e = CUBLASLT_EPILOGUE_DEFAULT;
   e = CUBLASLT_EPILOGUE_RELU;
+  e = CUBLASLT_EPILOGUE_GELU;
+  e = CUBLASLT_EPILOGUE_GELU_AUX;
```

Comment:
GELU_AUX is not as same as GELU.
According to CUBLAS doc, GELU_AUX  is
```
Apply GELU point-wise transform to the results (x := GELU(x)). This epilogue mode outputs GELU input as a separate matrix (useful for training). See CUBLASLT_MATMUL_DESC_EPILOGUE_AUX_POINTER of [cublasLtMatmulDescAttributes_t](https://docs.nvidia.com/cuda/cublas/index.html#cublasltmatmuldescattributes-t).
```
So you need update the implementation to copy the GELU input to EPILOGUE_AUX_POINTER